src/FixedDataTableCell.js

```diff
@@ -62,6 +62,12 @@ var FixedDataTableCell = createReactClass({
     rowIndex: PropTypes.number.isRequired,
 
     /**
+     * The data associated with the current row that will be passed to 
+     * `cellRenderer` to render.
+     */
+    rowData: PropTypes.object,
```

Comment:
Agreed, in my case I expect an object to be returned, though `PropTypes.any` does seem to be more appropriate in general.